ads/google/page-level-experiments.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Only question is whether this should continue to live in the ads/google hierarchy or just move to the src/ hierarchy?  Perhaps @lannka has an opinion?  (History: When I wrote the traffic-experiments.js code in the first place, we debated whether it was better to make it local to A4A or shared.  At that time, it was decided to keep it local until there was a need to expand it.  This is a clear need to expand, at least to Google scope.  Do we foresee that it could be valuable to make this available to other clients as well, and so move to src/?)